src/main/assembly/dist/cfg/datamanager.properties

```diff
@@ -0,0 +1,6 @@
+#
+# EASY datamanager configuration file
+# Unix-user
+#
+
+givennamefirstletteroffamilyname = userXXX givenname.familyname@dans.knaw.nl
```

Comment:
if you change this, implement Command.validDatamanager as follows:

```scala
datamanager.forall(dm => cfgDatamanagers.properties.getKeys.asScala
  .find(key => key.startsWith(dm))
  .exists(_ => true))
```

Reply:
I left this as it is now because in datamanagers.properties there are only two values given for a datamanager (unix-user):  Easy userid and email.  I think that it is clearer if these two values are on one line for each datamanager.

Reply:
as mentioned yesterday, this is not in sync with the pattern we started in other projects like `easy-ingest-flow` and `easy-sword2`. Maybe you have a point though. I leave this to @janvanmansum if he wants a specific format.

Reply:
I have now followed the same principle as in other projects